SECURITY.md

```diff
@@ -0,0 +1,86 @@
+<!--
+     Copyright 2021, Data61, CSIRO (ABN 41 687 119 230)
+
+     SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Vulnerability disclosure policy
+
+Security is a core value of the seL4 Foundation. If you believe you have
+found a security vulnerability in seL4, we ask you to work with us to
+resolve it according to principles of responsible disclosure. This
+policy outlines what we ask of you, and what you can expect of us.
+
+## Scope
+
+This policy currently applies to the most recent released versions, and
+the heads of the default branches of the software in the following seL4
+Foundation repositories:
+- https://github.com/seL4/seL4
+- https://github.com/seL4/seL4
+- https://github.com/seL4/seL4
+
+## What we ask of you
+
+- Report any vulnerabilities you discover as soon as you can, using the
+  official channel below.
+- Provide enough detail to allow us to understand the vulnerability and
+  its impact.
+- Be patient with us if we have questions.
+- Allow us reasonable time to fix the vulnerability before you disclose
```

Comment:
Google Project Zero gives 90 day. Should we give a time frame here also?

Reply:
I don't think we should commit to a time to fix. We do commit to a time to respond below.

Reply:
But maybe we should add a time frame of what we think is reasonable. 90 days might be way too short for a proof update, but it would be Ok for most code updates.

Reply:
I've added a bit to say we will usually aim for 90 days, but sometimes will need significantly longer.

Reply:
That looks good to me!